client/src/index.tsx

```diff
@@ -1,16 +1,13 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
 
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root') as HTMLElement,
-);
+const container = document.getElementById('root');
+const root = createRoot(container!);
```

Comment:
hi @junsupark94, thanks for getting this fix in.  I had to look up the postfix bang operator, so good TIL for me.  Posting the link here in case someone else is curious - https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci